workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -32,7 +32,21 @@ barcodes = ['10Xv2': '737K-august-2016.txt',
             '10Xv3.1': '3M-february-2018.txt']
 
 // supported single cell technologies
-tech_list = ['10Xv2', '10Xv3', '10Xv3.1'] 
+tech_list = barcodes.keySet()
+
+// file paths
+index_path = "${params.ref_dir}/${params.index_dir}/${index_names_map[params.index_type]}"
+index_prefix = "${params.ref_dir}/${params.annotation_dir}"
+t2g_path = "${index_prefix}/${t2g_map[params.index_type]}"
```

Comment:
I had actually meant to leave this in the workflow block, just higher up, but I think either place is fine.